### PR TITLE
Better fonts on headers and body

### LINKED ISSUE #115: Use the right fonts
## Acceptance criteria

- [ ] The fonts used for the body are correct (Rubik)
- [ ] The fonts used for headers are correct (Adelle)
- [ ] The fonts used for other headers are correct (Ludicrous)

## Implementation notes & questions

@undividual what are the fonts that we should be using?

### PR DESCRIPTION
**Pull request by ivan-kocienski-gfsc**
Fixes #115 

## Description

- Body uses Rubik
- Headers use Adelle

@geeksforsocialchange/developers

### CODE REVIEW

**Review by katjam — COMMENTED** [review]
This looks good. I think there was one more font that Emma added to dropbox too?

File: src/Page/Guide/View.elm [hunk 1]
```diff
@@ -54,3 +55,8 @@ viewRelatedGuideTeasers guideList =
 
     else
         text ""
+
+guideTitle: List Style
+guideTitle =
+    [ fontFamilies [ "Ludicrous", "serif" ]
```
Comment (katjam) [inline src/Page/Guide/View.elm:61]: Could argue for this to go in global with the other font declarations but f it only ever gets used here then this is a good home.

File: index.html [hunk 2]
```diff
@@ -6,6 +6,30 @@
...
+
+    <!-- Rubik font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Rubik:wght@300&display=swap"
+      rel="stylesheet"
+    />
+
+    <!-- ludicrous font -->
```
Comment (katjam) [inline index.html:25]: We might have wanted this with `src/css` but the other fonts import here - so I think this makes sense.

**Review by katjam — APPROVED** [review]
Beautiful!

File: src/Page/Guide/View.elm (hunk 1 above)
Comment (ivan-kocienski-gfsc) [reply src/Page/Guide/View.elm:61]: My thinking is that this is some placeholder code to indicate how this particular font can be applied to a title. As it only appears on one page on the site then its okay to have it locally in one view, I think.

File: index.html (hunk 2 above)
Comment (ivan-kocienski-gfsc) [reply index.html:25]: Wasn't sure if it should be in
- reset.css
- its own css file in the css folder (then also included in index.html)
- tucked in the index like this

So yeah I put it next to all the other font imports so they're all in the same place at least.

**Pull request merged**